<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAboutDialog</class>
 <widget class="QDialog" name="QtAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>563</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>[file_info] version [version]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_copyright">
     <property name="text">
      <string>Copyright (C) [years] [author]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_author">
     <property name="text">
      <string>Programmed by [author]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_date">
     <property name="text">
      <string>[date]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_download">
     <property name="text">
      <string>[file_info] can be downloaded from</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_url">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_licence">
     <property name="text">
      <string>Licenced under GPL 3.0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text_libraries">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text_whatsnew">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text_licence">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_build_date_time">
     <property name="text">
      <string>Source code built at [date][time]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_build_type">
     <property name="text">
      <string>[debug|release] build</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
